src/api/index.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const httpClient = axios.create( {
+  baseURL : process.env.BASE_URL,
```

Comment:
We can move the base url in constants file and import it from there

Reply:
Hey @samarpan1738! So, these environment variables can be added inside an `.env` file with `VUE_APP` as a prefix to each key, and then can be later used in the whole app. 

So, answering to your question, the base URL has already been added to the env file and therefore we are able to access it using `process.env`. 
@SBagaria2710 Can we add an example_env file in the project to make it clear for new devs?